src/param.c

```diff
@@ -43,17 +43,19 @@ void set_param_defaults(void)
     sprintf(temp_name, "TEMP_%c%c", 'A' + id/10, 'A' + id%10);
     init_param(id, temp_name, id);
   }
-  init_param(PARAM_BOARD_REVISION, "BOARD_REV", 5);
+  init_param(PARAM_BOARD_REVISION, "BOARD_REV", 4);
+
+  init_param(PARAM_BAUD_RATE, "BAUD_RATE", 921600);
 
   init_param(PARAM_SYSTEM_ID, "SYS_ID", 1);
   init_param(PARAM_STREAM_HEARTBEAT_RATE, "STRM_HRTBT", 1);
-  init_param(PARAM_STREAM_IMU_RATE, "STRM_IMU", 100);
+  init_param(PARAM_STREAM_IMU_RATE, "STRM_IMU", 500);
```

Comment:
I haven't had any problems at 500 Hz. Like I said above, 1000 Hz was doing great, simultaneously streaming commands at full rate, baro, differential pressure, and a `named_value_int` at something like 100 Hz.  I think we have a lot more bandwidth on this new system.  I think it would be a good idea to try to push the limits and see where stuff starts to break, but it's certainly not at 500 Hz.